vorbisgain/0009-hardening.patch

```diff
@@ -0,0 +1,13 @@
+Description: Allow build with dh compat level 9 (debian build flags)
+Forwarded: no
+--- a/misc.c
++++ b/misc.c
+@@ -56,7 +56,7 @@
+     vfprintf(stderr, message, args);
+     va_end(args);
+ 
+-    fprintf(stderr, strerror(err_num));
++    fputs(strerror(err_num), stderr);
+     fprintf(stderr, "\n");

```

Comment:
I know you didn't write this but these patches are pretty lazy and low quality IMO.

This could be `fprintf("%s\n", strerror(err_num));`